docs/ai/how-to/app-service-entra-auth.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Authenticate and Authorize App Service to Azure OpenAI using Microsoft Entra and the Semantic Kernel SDK"
+description: "Learn how to authenticate and authorize your app service application to an Azure OpenAI resource using Microsoft Entra managed identities and the Semantic Kernel SDK for .NET."
+author: haywoodsloan
+ms.topic: how-to
+ms.date: 04/19/2024
+zone_pivot_groups: azure-development-tool-set-two
+
+#customer intent: As a .NET developer, I want authenticate and authorize my App Service to Azure OpenAI using Microsoft Entra so that I can securely use AI in my .NET application.
+
+---
+
+# Authenticate and Authorize App Service to Azure OpenAI using Microsoft Entra and the Semantic Kernel SDK
+
+This article demonstrates how to use Microsoft Entra managed identities to authenticate and authorize an App Service application to an Azure OpenAI resource.
+
+Using a managed identity from Microsoft Entra allows your App Service application to easily access a protected Azure OpenAI resources without having to provision or rotate any secrets.
+
+The [Semantic Kernel SDK](/semantic-kernel/overview) enables you to easily implement Microsoft Entra authentication in your .NET application.
+
+## Prerequisites
+
+* An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* [.NET SDK](https://dotnet.microsoft.com/download/visual-studio-sdks)
+* [`Microsoft.SemanticKernel` NuGet package](https://www.nuget.org/packages/Microsoft.SemanticKernel)
+* [`Azure.Identity` NuGet package](https://www.nuget.org/packages/Azure.Identity)
+* [Create and deploy an Azure OpenAI Service resource](/azure/ai-services/openai/how-to/create-resource)
+* [Create and deploy a .NET application to App Service](/azure/app-service/quickstart-dotnetcore)
+
+## Add a managed identity to App Service
+
+### Add a system-assigned identity
+
+:::zone target="docs" pivot="azure-portal"
+
+1. Navigate to your app's page on the [Azure Portal](https://aka.ms/azureportal), scroll down to the **Settings** group.
+1. Select **Identity**.
+1. Within the **System assigned** tab, switch **Status** to On. Click **Save**.
```

Comment:
Yeah, it's a toggle switch where the labels are clickable: https://learn.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=portal%2Chttp#add-a-system-assigned-identity.

Thanks for the guidance on UI elements, based on that _Status_ should be italicized because it's a UI text but not clicked. And **On** should be bold because it's actually clickable.

I've also updated the formatting on a few other UI elements based on this info.